tardis/montecarlo/struct.py

```diff
@@ -0,0 +1,190 @@
+import os
+from ctypes import *
+
+import numpy as np
+from tardis import __path__ as path
+
+test_path = os.path.join(path[0], 'montecarlo', 'test_montecarlo.so')
+tests = CDLL(test_path)
+
+cmontecarlo_filepath = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
+cmontecarlo_methods = CDLL(cmontecarlo_filepath)
+
+
+class RPacket(Structure):
+    _fields_ = [
+        ('nu', c_double),
+        ('mu', c_double),
+        ('energy', c_double),
+        ('r', c_double),
+        ('tau_event', c_double),
+        ('nu_line', c_double),
+        ('current_shell_id', c_int64),
+        ('next_line_id', c_int64),
+        ('last_line', c_int64),
+        ('close_line', c_int64),
+        ('recently_crossed_boundary', c_int64),
+        ('current_continuum_id', c_int64),
+        ('virtual_packet_flag', c_int64),
+        ('virtual_packet', c_int64),
+        ('d_line', c_double),
+        ('d_electron', c_double),
+        ('d_boundary', c_double),
+        ('d_cont', c_double),
+        ('next_shell_id', c_int64),
+        ('status', c_int),
+        ('id', c_int64),
+        ('chi_th', c_double),
+        ('chi_cont', c_double),
+        ('chi_ff', c_double),
+        ('chi_bf', c_double)
+    ]
+
+    def __init__(self, **kwargs):
```

Comment:
`__init__` is inherited from `ctypes.Structure` so I had to write it within the struct declaration.